### PR TITLE
add var article

### PR DESCRIPTION
**Pull request by darlanmendonca**
I write a var article. I wish write articles to how work with differen types (string, number, array, objects, ...), in next article

after all, finally, articles about let and const

in this first, I

### CODE REVIEW

**Comment by darlanmendonca** [comment]
@khaosdoctor @OdineiRibeiro give a review pls

File: content/en/var.md [hunk 1]
```diff
@@ -0,0 +1,79 @@
...
+// invalid, var is a reserved word
+var var;
+```
+
+```js
+// valid, 
+var player_javasacript
+```
+
+Although symbols are allowed in variable names (_, $, @, entre outros), there are patterns of both Javascript,
```
Comment (odineiramone) [inline content/en/var.md]: Portuguesse and english on the same phrase

File: content/pt-br/var.md [hunk 2]
```diff
@@ -0,0 +1,78 @@
...
+
+Embora símbolos sejam permitidos no nome de variávies (_, $, @, entre outros), existem padrões tanto de Javascript, como de ferramentas que você provavelmente venha a usar. Mas em geral, os padrões mais importantes em Javascript são:
+
+- nome de variáveis são em letras minúsculas
+- se o nome da variável for composto, exemplo, user address, use camel case
+
+O camel case consiste em deixar a primeira letra das palavras subsequentes (não a primeira) em maíuscula
+
+```js
+// valid but not follow patterns
```
Comment (odineiramone) [inline content/pt-br/var.md]: English comment on pt-br file

File: content/pt-br/var.md [hunk 3]
```diff
@@ -0,0 +1,78 @@
...
+
+```js
+// valid and following patterns
+var userHeight;
+```
+
+Por último, um último padrão sobre declaração de variáveis. Se for declarar muitas ao mesmo tempo, o Javascript permite usar vírgula, mas não é o padrão.
+
+```js
+// valid but not follow patterns
```
Comment (odineiramone) [inline content/pt-br/var.md]: English comment on pt-br file

File: content/pt-br/var.md [hunk 4]
```diff
@@ -0,0 +1,78 @@
...
+var userName, userAddress, userHeight;
+```
+
+```js
+// valid and following patterns
```
Comment (odineiramone) [inline content/pt-br/var.md]: English comment on pt-br file

File: content/en/var.md (hunk 1 above)
Comment (darlanmendonca) [reply content/en/var.md]: :D, fixed, thanks

File: content/pt-br/var.md (hunk 2 above)
Comment (darlanmendonca) [reply content/pt-br/var.md]: fixed/translated

File: content/pt-br/var.md (hunk 3 above)
Comment (darlanmendonca) [reply content/pt-br/var.md]: fixed/translated

File: content/pt-br/var.md (hunk 4 above)
Comment (darlanmendonca) [reply content/pt-br/var.md]: fixed/translated

**Comment by darlanmendonca** [comment]
@OdineiRibeiro thanks man, I fixed the content not translated

**Comment by odineiramone** [comment]
:+1:

**Review by khaosdoctor — APPROVED** [review]
Made some changes in the english version, removed some portuguese and misplaced words. Good to Go

**Pull request merged**